.gitignore

```diff
@@ -54,3 +54,4 @@ idir/
 
 spu_wheel.name
 *.whl
+.vscode/settings.json
```

Comment:
revert this change please

Reply:
OK, done.

Reply:
Can you also add settings.json back?

Reply:
Is that OK now? I'm not very familiar with git yet :(

Reply:
perfect :P

This change looks fine to me. Let's wait for feedbacks from other reviewers.

Reply:
Thank you :)